package.json

```diff
@@ -3,11 +3,7 @@
   "version": "0.2.1",
   "scripts": {
     "make-pack": "npm run build && node ./scripts/pack-theme",
-    "build": "npm run build:sass",
-    "build:sass": "npm run build:sass:login",
-    "build:sass:hds": "npm-sass sass/hds/index.scss >",
-    "build:sass:shared": "npm-sass sass/shared/index.scss >",
-    "build:sass:login": "mkdirp helsinki/login/resources/css && npm run build:sass:hds -- helsinki/login/resources/css/hds.css && npm run build:sass:shared -- helsinki/login/resources/css/shared.css && npm-sass sass/login/index.scss > helsinki/login/resources/css/hs-login.css",
+    "build": "gulp",
```

Comment:
What's the benefit of running individual SCSS-to-CSS file transformations? The whole current build executes in less than half a second on my machine so just running the whole build at least doesn't waste much time.

The three individual CSS file builds could be separated to their own tasks. But Gulp build runs all three SCSS-to-CSS transformations in parallel so transforming all three files or just a single one probably doesn't have much time difference.

Various build commands _can_ be added to `package.json` even if Gulp is used for the various steps. The Gulp configuration can be split into steps and those steps can be revealed so that they can be run individually.

Reply:
What is the benefit of using gulp to build css files? Gulp is not commonly used in Helsinki projects, so at least to me it is a new tool to do old things in a different way. Theme.properties need to be modified with gulp of course. Or with node or a shell script.

Reply:
I first started to write logic for achieving these build steps without any additional tools, just JavaScript. It quickly started to look like I'm writing a build tool so instead of writing my own, I figured an existing one should be used. Gulp is the one tool I knew of. Any other build tool can be used also.

Reply:
This PR is also currently missing the "cache bursting" of JavaScript files so some more build steps need to be added too.

Reply:
Or JavaScript files needs some "handling", regardless of how this is achieved.

Reply:
Gulp is fine. Just wondered should it just do the cache busting and leave old build process as it is.

Js files are listed like css files in theme.properties, so "?v=" can be added in the same way.

Reply:
> Gulp is fine. Just wondered should it just do the cache busting and leave old build process as it is.

I don't know what's the benefit. Is the need that it should be possible to transform individual CSS files by running `npm run ...` commands? If that's the case, separate tasks can be added to the Gulp build. So there can be `gulp sass`, `gulp sass:hds` etc. build tasks. And there can be their corresponding npm scripts in `package.json`, e.g.

```json
[--]
"build": "gulp",
"build:sass": "gulp sass",
"build:sass:hds", "gulp sass:hds",
[--]
```

> Js files are listed like css files in theme.properties, so "?v=" can be added in the same way.

Yep, that's the plan.